src/Browscap/Browscap.php

```diff
@@ -1,16 +1,27 @@
 <?php
 /**
- * Copyright (c) 1998-2014 Browser Capabilities Project
+ * Copyright (c) 1998-2017 Browser Capabilities Project
  *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
- * published by the Free Software Foundation, either version 3 of the
- * License, or (at your option) any later version.
+ * Permission is hereby granted, free of charge, to any person obtaining a
```

Comment:
I don't think we need the whole license in the headers here. Just `Refer to the LICENSE file distributed with this package.` should suffice please.